allwinner/h6_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2024 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package allwinner
+
+import (
+	"testing"
+)
+
+func TestGetH6SerializedPinSpecs(t *testing.T) {
+	if pins, err := getH6SerializedPinSpecs(); err != nil {
+		t.Error(err)
```

Comment:
Optional: if you used t.Fatal() here, you wouldn't have to use "else" here and line 22.

Reply:
fixed,